2-mandatory/1-writers.js

```diff
@@ -81,9 +90,15 @@ Exercise 3:
 */
 
 function logAliveWritersInTheirForties() {
-  // write your code here
+  writers.forEach(function(e){
+    if(e.age<=49 && e.age>=40 && e.alive === true){
```

Comment:
Can also be written as:
```suggestion
    if(e.age<=49 && e.age>=40 && e.alive){
```

Can you see the difference and why they are equal?